node/config/def.go

```diff
@@ -12,9 +12,10 @@ import (
 
 // Common is common config between full node and miner
 type Common struct {
-	API    API
-	Libp2p Libp2p
-	Pubsub Pubsub
+	API        API
+	Libp2p     Libp2p
+	Pubsub     Pubsub
+	Blockstore Blockstore
```

Comment:
I'd put this in the FullNode struct, and call `Chainstore` as we have some other blockstores

Reply:
ok, can do.

Reply:
done.